jupyter/4 - dbGap FHIR Controlled Access via RAS -  COPDGene.ipynb

```diff
@@ -0,0 +1,1058 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>28</th>\n",
+       "      <td>phs002409</td>\n",
+       "      <td>FHIR Test Study's ALPHA</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "           id                                              title\n",
+       "25  phs000178                     The Cancer Genome Atlas (TCGA)\n",
+       "12  phs000179            Genetic Epidemiology of COPD (COPDGene)\n",
+       "1   phs000209  Multi-Ethnic Study of Atherosclerosis (MESA) C...\n",
+       "30  phs000235  National Cancer Institute Cancer Genome Charac...\n",
+       "0   phs000614    Genomic Analysis of Pediatric Low Grade Gliomas\n",
+       "22  phs000748                    Multiple Myeloma CoMMpass Study\n",
+       "2   phs000794  Longitudinal Genome-Wide Analysis of Patients ...\n",
+       "8   phs000892             CPTAC Proteogenomic Confirmatory Study\n",
+       "27  phs000951  NHLBI TOPMed: Genetic Epidemiology of COPD (CO...\n",
+       "4   phs001054  Genomic Characterization of Pediatric Low-Grad...\n",
+       "3   phs001078                         Common Variant GWAS, GECCO\n",
+       "21  phs001175            CTSP: Clinical Trial Sequencing Project\n",
+       "6   phs001179  Foundation Medicine Adult Cancer Clinical Data...\n",
+       "5   phs001228  Gabriella Miller Kids First (GMKF) Pediatric R...\n",
+       "29  phs001287                                      CPTAC 3 Study\n",
+       "7   phs001315  GECCO: Detecting Common and Rare Genetic Loci ...\n",
+       "26  phs001374  VA APOLLO Project - Research for Precision Onc...\n",
+       "10  phs001415  Uncovering the Genetic Architecture of Colorec...\n",
+       "11  phs001436  Discovering the Genetic Basis of Human Neurobl...\n",
+       "9   phs001437    Pediatric Preclinical Testing Consortium (PPTC)\n",
+       "24  phs001444  Genomic Variation in Diffuse Large B Cell Lymp...\n",
+       "23  phs001486  NCI Cancer Model Development for the Human Can...\n",
+       "13  phs001524  The Genetic Basis of Aggressive Prostate Cance...\n",
+       "15  phs001554  Detection of Colorectal Cancer Susceptibility ...\n",
+       "16  phs001611                 Pancreas Cancer Organoid Profiling\n",
+       "17  phs001628              Clinical Crenolanib Resistance in AML\n",
+       "14  phs001657  Functional Genomic Landscape of Acute Myeloid ...\n",
+       "20  phs001713                               LCCC 1108 (UNCseqTM)\n",
+       "19  phs001714  Kids First: Genomic Analysis of Treatment Fail...\n",
+       "18  phs001738  Kids First: Familial Predisposition to Hematop...\n",
+       "28  phs002409                            FHIR Test Study's ALPHA"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sdict = []\n",
```

Comment:
I think a list comprehension is appropriate here.
```
sdict = [{'id':s['id'], 'title':s['title']} for s in studies]
```

Reply:
list comprehension was consciously avoided here and in other places. Useful for some things not for others.